src/halide_ir.fbs

```diff
@@ -713,6 +713,7 @@ table Func {
     trace_stores: bool = false;
     trace_realizations: bool = false;
     trace_tags: [string];
+    no_profiling: bool = false;
```

Comment:
Probably need to bump the serialization minor version number

Reply:
I'm actually clueless where to find that number.

Reply:
I'm a bit confused too. @TH3CHARLie ?

Reply:
@TH3CHARLie looking at those inline comments, suggest that I shouldn't touch anything, as we are still at "unstable"? Do I bump the patch version instead?

Reply:
yeah I believe so

Reply:
@abadams @TH3CHARLie It all looks suspicious, as the Patch version was at 0 (zero). Serialization version 18.0.0 seems suspicious, but maybe is what we are after? I just bumped it to 18.0.1.

Reply:
It was at zero since we recently had a release and introduced this version numbering system and hasn't added anything new to the format, I don't understand what's suspicious here.

Reply:
Makes sense! Thanks!